write-fonts/src/tables/variations.rs

```diff
@@ -116,8 +116,12 @@ impl PackedDeltas {
             (MIN..=MAX).contains(&val)
         }
 
-        fn count_leading_zeros(slice: &[i16]) -> usize {
-            slice.iter().take_while(|v| **v == 0).count()
+        fn count_leading_zeros(slice: &[i16]) -> u8 {
+            slice
+                .iter()
+                .take(u8::MAX.into())
```

Comment:
iirc, the max run length doesn't change here, and should be 64; the run length is stored in the low 6 bits of the control byte; [see the docs here](https://learn.microsoft.com/en-us/typography/opentype/spec/otvarcommonformats#packed-deltas).

Reply:
https://github.com/googlefonts/fontations/pull/356